x-pack/legacy/plugins/siem/public/components/timeline/styles.tsx

```diff
@@ -264,16 +264,16 @@ export const EventsHeadingTitleButton = styled.button.attrs({
 `;
 EventsHeadingTitleButton.displayName = 'EventsHeadingTitleButton';
 
-export const EventsHeadingTitleSpan = styled.span.attrs({
+export const EventsHeadingTitleSpan = styled.span.attrs(props => ({
   className: 'siemEventsHeading__title siemEventsHeading__title--notAggregatable',
-})`
+}))`
   min-width: 0;
 `;
 EventsHeadingTitleSpan.displayName = 'EventsHeadingTitleSpan';
 
-export const EventsHeadingExtra = styled.div.attrs({
-  className: 'siemEventsHeading__extra',
-})<{ className?: string }>`
+export const EventsHeadingExtra = styled.div.attrs(props => ({
+  className: props.className || 'siemEventsHeading__extra',
```

Comment:
Is this change indicating that the `className` should be whatever is passed to this component as `className` OR `siemEventsHeading__extra`? If so, to keep the same functionality we have today, we need to change it so that it outputs both classes (`siemEventsHeading__extra` and any `className`s that may be provided to the component), not just one or the other.

Reply:
Changed, Thank you @MichaelMarcialis